csrc/quantization/quantization_kernels.cu

```diff
@@ -0,0 +1,477 @@
+/*
```

Comment:
Please add the link to the original code in the AWQ repo for the code copied from their repo.